Tech-Journal-ConsoleApp/Tech-Journal-ConsoleApp/SendEmail.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.IO;
+using System.Net;
+using System.Net.Mail;
+using System.Text;
+
+namespace Tech_Journal_ConsoleApp
+{
+    class SendEmail
+    {
+        public const string Path = "c:\\temp\\appsettings.txt";
+
+        private bool _validEmail;
+        private string _toThisEmailAddress;
+        private string _emailUsername;
+        private string _emailPassword;
+
+
+        public SendEmail()
+        {
+            _validEmail = false;
+        }
+        public void ValidateEmailSettings()
+        {
+            if (!File.Exists(Path))
```

Comment:
I think you can split this method into seperate methods.
A validate method to check the information inside this class is valid (Is there an email and password set).
A 'Populate' method to handle populating the file.
A 'Read' method to handle reading the information on the file.